plugins/imjournal/imjournal.c

```diff
@@ -93,6 +94,7 @@ static struct cnfparamdescr modpdescr[] = {
 	{ "defaultseverity", eCmdHdlrSeverity, 0 },
 	{ "defaultfacility", eCmdHdlrString, 0 },
 	{ "usepidfromsystem", eCmdHdlrBinary, 0 },
+	{ "usepid", eCmdHdlrString, 0 },
```

Comment:
I notice that was wrong previously as well, but it would be great if you could remove the extra/invalid comma.